# HG changeset patch
# User Ben Abbott  <bpabbott@mac.com>
# Date 1327258035 18000
#      Sun Jan 22 13:47:15 2012 -0500
# Node ID 27abe77158d6142f80e73d9df2210f770c1394aa
# Parent  48cb466f1418d8e1f7f21ab971cddea44150868e
Changes to allow plot demos to be converted and run by Matlab.

* text.m, waitbar.m

diff --git a/scripts/plot/text.m b/scripts/plot/text.m
--- a/scripts/plot/text.m
+++ b/scripts/plot/text.m
@@ -126,17 +126,18 @@ function h = text (varargin)
 
 endfunction
 
 
 %!demo
 %! clf;
 %! ha = {'left', 'center', 'right'};
 %! va = {'bottom', 'middle', 'top'};
-%! x = y = [0.25 0.5 0.75];
+%! x = [0.25 0.5 0.75];
+%! y = x;
 %! for t = 0:30:359;
 %!   for nh = 1:numel(ha)
 %!     for nv = 1:numel(va)
 %!       text (x(nh), y(nv), 'Hello World', ...
 %!             'rotation', t, ...
 %!             'horizontalalignment', ha{nh}, ...
 %!             'verticalalignment', va{nv});
 %!     end
diff --git a/scripts/plot/waitbar.m b/scripts/plot/waitbar.m
--- a/scripts/plot/waitbar.m
+++ b/scripts/plot/waitbar.m
@@ -162,17 +162,17 @@ endfunction
 %!   waitbar (i, h, 'please don''t be impatient');
 %! end
 %! close (h);
 
 %!demo
 %! h1 = waitbar (0, 'Waitbar #1');
 %! h2 = waitbar (0, 'Waitbar #2');
 %! h2pos = get (h2, 'position');
-%! h2pos(1) += (h2pos(3) + 50);
+%! h2pos(1) = h2pos(1) + (h2pos(3) + 50);
 %! set (h2, 'position', h2pos);
 %! pause (0.5);
 %! for i = 1:4
 %!   waitbar (i/4, h1);
 %!   pause (0.5);
 %!   waitbar (i/4, h2);
 %!   pause (0.5);
 %! end
